gateway-ha/src/test/java/io/trino/gateway/ha/security/TestLbTokenUtil.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.gateway.ha.security;
+
+import com.auth0.jwt.JWT;
+import com.auth0.jwt.interfaces.DecodedJWT;
+import io.trino.gateway.ha.config.SelfSignKeyPairConfiguration;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.junit.jupiter.api.TestInstance.Lifecycle;
+
+import java.util.List;
+import java.util.Optional;
+
+import static java.util.Objects.requireNonNull;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@TestInstance(Lifecycle.PER_CLASS)
+public class TestLbTokenUtil
+{
+    private final String idToken = "eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJ0ZXN0Iiwic3ViIjoidGVzdCIsImF1ZCI6InRlc3QuY29tIn0.YVdTAuEnn-UHsZf5AKA8avaBTLQvdQ3Dd7LeaXKSjlq0atRcmp_IzE8vnJYplYwBs6VNsGpgrfeIMXUsZFKTcfIBinAh05Fj5M7zXVoOPLEdYH5QJ_sfe5-KFPuJs32epC7GE50-umLKTHDZ14dfiXUNK4yAE0bmiNh1lhKcEP35ZAaBt8ldngXJ7BlNbpLpS30-Z4vA6Nht-WjIlluVHd3B5LCJO5FKRcjpuV7E_XZID9c8p0_7Wl7yEvaQB0yJd-VKgT2MQAZse17YqBTb3ix51F5N05s98w7w5zX8NI_afp97YWwNPKTN0dVZFVF8abZ-35O2CAlM0hzbt7NKdw";
```

Comment:
Could you use `JWT.create()` to build the token in `setup()`? That would make these tests easier to understand and maintain in the future if new features are desired.